examples/loadbalancers/http-with-sticky-sessions.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
<img width="1058" alt="screen shot 2018-03-18 at 6 52 02 pm" src="https://user-images.githubusercontent.com/12699319/37572164-88bf7866-2add-11e8-9204-cb896e669b32.png">

Reply:
Is this the behavior?

Reply:
Yeah! Just putting it here to show what the sticky session config will look like based on the example

Reply:
I'm happy to see it's working as expected! :blush:

Reply:
Same! Thanks again for the contribution :)